index.html

```diff
@@ -19,12 +19,14 @@ <h1>Lisanne Kelly Ardern</h1>
 
     <main>
       <figure>
-        <img src="images/Minca.JPG" alt="" />
-        <figcaption>Me in Minca, Colombia in 2019</figcaption>
+        <img src="images/IMG_1059optsw-min.jpg" alt="" />
+        <figcaption>
+          Hello! My name is Lisanne and I am learning coding.
+        </figcaption>
       </figure>
 
       <div class="column">
```

Comment:
I'd switch the `div` and the `sections` and add a name which is not as generic. So the structure would be:
```suggestion
      <section class="about">
        <div class="about__interests">...</div>
        <div class="about__skills">...</div>
      </section>
```